README.md

```diff
@@ -467,6 +467,7 @@
 * [Vector Logic](https://www.vector-logic.com/blog/posts)
 * [Visuality](https://www.visuality.pl/posts)
 * [Wonolo](https://engineeringblog.wonolo.com/tag/ruby)
+* [Decode Fix](https://decodefix.com/tag/ruby/)
```

Comment:
@WALEED-QADEER please, insert line in alphabetical order (after `Datarockets`)

Reply:
Sure